contrib/android-configure.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+set +x
+
+default_abis="armeabi-v7a arm64-v8a x86 x86_64"
```

Comment:
We could probably drop x86 from this.  ISTR there were some issues with it on the wallet, and so we dropped it and no one noticed because there is virtual no x86 android hardware in the wild.

Reply:
the reason i am doing this cleanup is in preparation for potentially doing fdroid packaging. i do not know what the fdroid platforms that are supported